lib/features/children/details/pages/child_details_page.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/app/routes/routes.dart';
+import 'package:givt_app/core/enums/amplitude_events.dart';
+import 'package:givt_app/features/children/details/cubit/child_details_cubit.dart';
+import 'package:givt_app/features/children/details/widgets/child_details_item.dart';
+import 'package:givt_app/features/children/details/widgets/child_giving_allowance_card.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:go_router/go_router.dart';
+
+class ChildDetailsPage extends StatelessWidget {
+  const ChildDetailsPage({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final locals = AppLocalizations.of(context);
+    return BlocBuilder<ChildDetailsCubit, ChildDetailsState>(
+      builder: (context, state) {
+        return Scaffold(
+          appBar: AppBar(
+            actions: [
+              BackButton(
+                onPressed: () {
+                  context.pop();
+                  AnalyticsHelper.logEvent(
+                    eventName: AmplitudeEvents.backClicked,
+                  );
+                },
+              ),
+              const Spacer(),
```

Comment:
Does this work on a smaller screen? Or isn't overflowing?

Reply:
Do you see any issues with Spacer here? not sure how it can lead to overflow.. The layout itself also quite flexible. Not sure where is the issue here?